Dockerfile

```diff
@@ -9,16 +9,11 @@ RUN apt update -y -q && \
         ca-certificates \
         wget \
         gnupg2 \
-        git && \
-    curl -s https://storage.googleapis.com/golang/go1.18.3.linux-amd64.tar.gz| tar -v -C /usr/local -xz && \
```

Comment:
We need to keep this line, otherwise we won't have the `go` binary available.

Reply:
Should this just be replaced with `golang`?

Reply:
Let's keep using the binary distribution. We could also update to 1.19.1 while at it.